packages/web-components/src/radio/radio.styles.ts

```diff
@@ -0,0 +1,173 @@
+import { css } from '@microsoft/fast-element';
+import { display } from '@microsoft/fast-foundation';
+import {
+  borderRadiusMedium,
+  colorCompoundBrandForeground1,
+  colorCompoundBrandForeground1Pressed,
+  colorCompoundBrandStroke,
+  colorCompoundBrandStrokeHover,
+  colorCompoundBrandStrokePressed,
+  colorNeutralForeground1,
+  colorNeutralForeground2,
+  colorNeutralForeground3,
+  colorNeutralForegroundDisabled,
+  colorNeutralStrokeAccessible,
+  colorNeutralStrokeAccessibleHover,
+  colorNeutralStrokeAccessiblePressed,
+  colorStrokeFocus1,
+  colorStrokeFocus2,
+  fontFamilyBase,
+  fontSizeBase300,
+  fontWeightRegular,
+  lineHeightBase300,
+  spacingHorizontalS,
+  spacingHorizontalXS,
+  spacingVerticalS,
+  spacingVerticalXS,
+} from '../theme/design-tokens.js';
+
+/** Radio styles
+ * @public
+ */
+export const styles = css`
+  ${display('inline-flex')}
+
+  :host {
+    align-items: center;
+    flex-direction: row;
+    outline: none;
+    position: relative;
+    user-select: none;
+    cursor: pointer;
+  }
+
+  .label {
+    color: ${colorNeutralForeground3};
+    padding: ${spacingVerticalS} ${spacingHorizontalS} ${spacingVerticalS} ${spacingHorizontalXS};
+    font-family: ${fontFamilyBase};
+    font-size: ${fontSizeBase300};
+    font-weight: ${fontWeightRegular};
+    line-height: ${lineHeightBase300};
+    cursor: pointer;
+  }
+  .label__hidden {
+    display: none;
+    visibility: hidden;
+  }
+  .control {
+    position: relative;
+    display: inline-block;
+    width: 16px;
+    height: 16px;
+    border-radius: 50%;
+    border: 1px solid ${colorNeutralStrokeAccessible};
+    margin: ${spacingVerticalS} ${spacingHorizontalS};
+    top: 0;
+    left: 0;
```

Comment:
This won't localize, we can either use `inset-inline` which support only goes back to Safari 14 or figure out a way to not position it using `inset`

Reply:
I'm actually curious why we need top and left here at all.